scripts/script.c

```diff
@@ -0,0 +1,7 @@
+#include<stdlib.h>
+
+int main(void)
+{
+    system("echo /usr/bin/ls");
```

Comment:
The code is unsafe. Fix it.

Reply:
Please specify which line of code is actually unsafe.
The intended unsafe code is the system() function. 
A self-respecting maintainer should be more precise in such matter. In the future please try to keep a standard of precision in your further endeavors lest  you be removed from the maintainers list of the repo

Reply:
I see your point, but, unfortunately, your mother.